observability/logging/logging-6.1/log6x-about.adoc

```diff
@@ -0,0 +1,65 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="log6x-about"]
+= Logging 6.1
+:context: logging-6x
+
+toc::[]
+
+The `ClusterLogForwarder` custom resource (CR) is the central configuration point for log collection and forwarding.
+
+== Inputs and Outputs
+
+Inputs specify the sources of logs to be forwarded. Logging provides built-in input types: `application`, `infrastructure`, and `audit`, which select logs from different parts of your cluster. You can also define custom inputs based on namespaces or pod labels to fine-tune log selection.
+
+Outputs define the destinations where logs are sent. Each output type has its own set of configuration options, allowing you to customize the behavior and authentication settings.
+
+
+== Receiver Input Type
+The receiver input type enables the Logging system to accept logs from external sources. It supports two formats for receiving logs: `http` and `syslog`.
+
+The `ReceiverSpec` defines the configuration for a receiver input.
+
+== Pipelines and Filters
+
+Pipelines determine the flow of logs from inputs to outputs. A pipeline consists of one or more input refs, output refs, and optional filter refs. Filters can be used to transform or drop log messages within a pipeline. The order of filters matters, as they are applied sequentially, and earlier filters can prevent log messages from reaching later stages.
+
+== Operator Behavior
+
+The Cluster Logging Operator manages the deployment and configuration of the collector based on the `managementState` field:
```

Comment:
Is it worth clarifying that the field is in the CLF?    
ex.  ".... based on the `managementState` field of the `ClusterLogForwarder` resource: "